helm/values.yaml

```diff
@@ -0,0 +1,71 @@
+# coder -- Primary configuration for `coder server`.
+coder:
+  # NOTE: this is currently not used as coder v2 does not support high
+  #       availability yet.
+  # # coder.replicaCount -- The number of Kubernetes deployment replicas.
+  # replicaCount: 1
+
+  # coder.image -- The image to use for Coder.
+  image:
+    # coder.image.repo -- The repository of the image.
+    repo: "ghcr.io/coder/coder"
+    # coder.image.tag -- The tag of the image, defaults to {{.Chart.AppVersion}}
+    # if not set.
+    tag: ""
+    # coder.image.pullPolicy -- The pull policy to use for the image. See:
+    # https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
+    pullPolicy: IfNotPresent
+
+  # coder.env -- The environment variables to set for Coder. These can be used
+  # to configure all aspects of `coder server`. Please see `coder server --help`
+  # for information about what environment variables can be set.
+  #
+  # Note: The following environment variables are set by default and cannot be
+  # overridden:
+  # - CODER_ADDRESS: set to 0.0.0.0:80 and cannot be changed.
+  # - CODER_TLS_ENABLE: set if tls.secretName is not empty.
+  # - CODER_TLS_CERT_FILE: set if tls.secretName is not empty.
+  # - CODER_TLS_KEY_FILE: set if tls.secretName is not empty.
+  env:
+    - name: CODER_ACCESS_URL
+      value: "https://coder.example.com"
+    #- name: CODER_PG_CONNECTION_URL
+    #  value: "postgres://coder:password@postgres:5432/coder?sslmode=disable"
```

Comment:
What is our recommended setup in this deployment model?

It'd be awesome if you wrote docs as part of this PR (or in a future one, but soon after this). I'm not entirely sure what happens after I `helm install` this. I believe it just fails due to no PostgreSQL cluster?

Reply:
Made an issue about writing documentation: #3224

Reply:
at the moment it would just fail unless you specified a postgres deployment URL like that